Tutorials/intro-to-timing/timing_in_qua.py

```diff
@@ -35,44 +35,26 @@
 samples = job.get_simulated_samples()
 
 plt.figure(1)
-
-plt.subplot(2, 1, 1)
-samples.con1.plot("1")
-plt.xlabel("Time[ns]")
-plt.ylabel("qubit")
-plt.title("example 1 - Two pulses from different elements")
-plt.subplot(2, 1, 2)
-samples.con1.plot("2")
-plt.xlabel("Time[ns]")
-plt.ylabel("resonator")
-
-plt.subplots_adjust(left=0.1, bottom=0.1, right=0.9, top=0.9, wspace=0.4, hspace=0.4)
-
+samples.con1.plot()
+plt.legend(["qubit", "resonator"])
 
 # Simulate and plot 2nd example
 job = qmm.simulate(config, example2, SimulationConfig(int(1000)))  # in clock cycles, 4 ns
 samples = job.get_simulated_samples()
+plt.title("example 1 - Two pulses from different elements")
```

Comment:
This is in the wrong place and is not being shown, is that on purpose?

Reply:
Indeed.. it was in the wrong place but it is not being shown on purpose since the title is actually written in the sentence above each figure...

Reply:
Cool, then move all of the titles to the correct place and then we can merge